### PR TITLE
Client form view

### PR DESCRIPTION
**Pull request by ismith14**
Let me know if I need to change or add anything to the form.
Create:
client form create
Read (Allows user to update)
client form read
Update:
client form update

### CODE REVIEW

File: src/View/clientForm/clientFormView.java [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+            contactText.setText("Phone");
+            addressText.setText("1234 St.");
+            cityText.setText("Baltimore");
+            stateText.setText("MD");
+            zipText.setText("11111");
+            typeText.setText("Individual");
+
+            clientButton = new JButton("Update Client");
+        }
+        else if(action.equals("Create")) {
```
Comment (Cwagne17) [inline src/View/clientForm/clientFormView.java:86]: There should be another action for "Update"

File: src/View/clientForm/clientFormView.java [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+        {
+            nameText.setText("name");
+            emailText.setText("1234@email.com");
+            phoneText.setText("111-111-1111");
+            contactText.setText("Phone");
+            addressText.setText("1234 St.");
+            cityText.setText("Baltimore");
+            stateText.setText("MD");
+            zipText.setText("11111");
+            typeText.setText("Individual");
```
Comment (Cwagne17) [inline src/View/clientForm/clientFormView.java:80]: setText looks good, when we connect to the Model we will change this. If you could add a comment on ln 55 so we know to fix this when we are ready to connect to the model

File: src/View/clientForm/clientFormView.java [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+
```
Comment (Cwagne17) [inline src/View/clientForm/clientFormView.java]: use .setEditable(false); on each of the JTextFields for Read that way the input is immutable

File: src/View/clientForm/clientFormView.java [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+        clientEmail = new JLabel("Email : ");
+        clientPhone = new JLabel("Phone Number: ");
+        clientContact = new JLabel("Preferred Contact: ");
+        clientAddress = new JLabel("Address: ");
+        clientCity = new JLabel("City: ");
+        clientState = new JLabel("State: ");
+        clientZIP = new JLabel("ZIP: ");
+        clientType = new JLabel("Type: ");
+
+        nameText = new JTextField(20);
```
Comment (Cwagne17) [inline src/View/clientForm/clientFormView.java:51]: if the int is the max length of a text field we should do 45 for each of these.

**Review by Cwagne17 — CHANGES_REQUESTED** [review]
Looks good, just a few changes. Could you attach the screen shot to the PR comments?

File: src/View/clientForm/clientFormView.java (hunk 4 above)
Comment (ismith14) [reply src/View/clientForm/clientFormView.java:51]: The int is the width of the text field.

File: src/View/clientForm/clientFormView.java (hunk 1 above)
Comment (Cwagne17) [reply src/View/clientForm/clientFormView.java:86]: Please add another action for else if(action.equals("Update"))

**Review by Cwagne17 — CHANGES_REQUESTED** [review]

**Review by Cwagne17 — APPROVED** [review]

**Pull request closed without merging**